### DISCUSSION TITLE
Why HTMX over Livewire

### DISCUSSION THREAD

**Discussion by mokhosh** [vitodeploy/vito#126] [General]
Just saw the release notes in my feed and had to jump here to ask what made you decide to migrate from Livewire to HTMX?
Thanks

**Comment by saeedvaziry** [comment]
In order to make the development easier and faster to understand the structure for the new contributors I decided to reduce the complexity and remove an extra layer.
Livewire in vito wasn’t handling any logic and was just passing the data to the Actions.
